app/_data/kong_versions.yml

```diff
@@ -133,6 +133,16 @@
     cassandra: "3.x.x"
     postgres: "9.5+"
     openresty: "1.13.6.2"
+-
+  release: "1.0"
+  version: "1.0"
+  luarocks_version: "1.0.0-0"
+  dependencies:
+    luajit: "2.1.0-beta2"
+    luarocks: "2.4.3"
+    cassandra: "3.x.x"
+    postgres: "9.5+"
+    openresty: "1.13.6.2"
```

Comment:
I'm not sure why yet but it appears this block is what is preventing the index from rendering.

Reply:
Still debugging but as far as I can tell, the compiler is breaking because of the 1.0 release/version number